opencti-platform/opencti-front/src/private/components/common/form/ObjectMembersField.tsx

```diff
@@ -0,0 +1,121 @@
+import { graphql } from 'react-relay';
+import makeStyles from '@mui/styles/makeStyles';
+import React, { FunctionComponent, useState } from 'react';
+import { Field } from 'formik';
+import { Theme } from '../../../../components/Theme';
+import { fetchQuery } from '../../../../relay/environment';
+import { useFormatter } from '../../../../components/i18n';
+import { ObjectMembersFieldSearchQuery$data } from './__generated__/ObjectMembersFieldSearchQuery.graphql';
+import AutocompleteField from '../../../../components/AutocompleteField';
+import ItemIcon from '../../../../components/ItemIcon';
+import { Option } from './ReferenceField';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  icon: {
+    paddingTop: 4,
+    display: 'inline-block',
+    color: theme.palette.primary.main,
+  },
+  text: {
+    display: 'inline-block',
+    flexGrow: 1,
+    marginLeft: 10,
+  },
+  autoCompleteIndicator: {
+    display: 'none',
+  },
+}));
+
+const objectMembersFieldSearchQuery = graphql`
+    query ObjectMembersFieldSearchQuery($search: String, $first: Int) {
+        members(search: $search, first: $first) {
+            edges {
+                node {
+                    id
+                    entity_type
+                    name
+                }
+            }
+        }
+    }
+`;
+
+interface OptionMember extends Option {
+  type: string;
+}
+
+interface ObjectMembersFieldProps {
+  name: string;
+  onChange?: (name: string, value: OptionMember) => void;
+  style?: Record<string, string | number>;
+  helpertext?: string;
```

Comment:
Why we have the properties onChange, style, helpertext ?
Seems to be not used

Reply:
yes they are not used for now, but since it's a generic field component I thought we still want to keep them. But I'm not sure what we should do when creating generic components.